setup.py

```diff
@@ -32,7 +32,7 @@ def package_data(pkg, root_list):
 
 setup(
     name='xblock-ooyala-player',
-    version='5.0.0',
+    version='4.1.3',
```

Comment:
on juniper we are using version number 5.0.0, so this should be bumped to 5 series